.vscode/launch.json

```diff
@@ -0,0 +1,4 @@
+{
+	"version": "0.2.0",
+	"configurations": []
+}
```

Comment:
🤔

Reply:
Maybe a `git rm .vscode/launch.json`?

Reply:
ok so we don't require this whole file..